lib/renderer/content-scripts-injector.js

```diff
@@ -23,6 +23,12 @@ const runContentScript = function (extensionId, url, code) {
   return compiledWrapper.call(this, context.chrome)
 }
 
+const runAllContentScript = (scripts, extensionId) => () => {
```

Comment:
It seems unnecessary to have a function returning a function, I would prefer a normal function and let the caller bind it:

```js
const runAllContentScript = (scripts, extensionId) => {
  ...
}

const fire = runAllContentScript.bind(window, script.js, extensionId)
```